RELEASES.md

```diff
@@ -86,31 +86,39 @@ project layouts.**
 
 ```bash
 python -m pip install \
-  --pre --find-links https://therock-nightly-python.s3.amazonaws.com/gfx94X-dcgpu/index.html \
+  --index-url https://d2awnip2yjpvqn.cloudfront.net/v2/gfx94X-dcgpu/ \
```

Comment:
I assume we'll get a nicer URL for this at some point? I don't really want to point users at such an obfuscated URL, even if it is faster and more stable... could maybe do it with a callout in the section above that we're registering a domain name or something.

Reply:
Yes, we definitely should get a nicer one.